src/Typography/Typography.component.tsx

```diff
@@ -139,14 +137,14 @@ const StyledTypography = (tag: TypographyTags) => styled(tag)<
 export const Typography = ({
     className,
     children,
-    tag = 'span',
+    as = 'span',
     color,
     ...props
-}: TypographyProps): JSX.Element =>
-    React.createElement(StyledTypography(tag), {
-        children,
-        tag,
-        $color: color,
-        className: classNames('anchor-typography', className),
-        ...props,
-    });
+}: TypographyProps): JSX.Element => (
+    <StyledTypography
+        className={classNames('anchor-typography', className)}
+        {...{ as, color, ...props }}
```

Comment:
`$color: color` keeps `color="text.base"` out of the dom.

you should be able to keep it out of this object spread and just do `$color={color}`

Reply:
Another artifact from my naive first SC.